opensrp-unicef-tunisia/build.gradle

```diff
@@ -371,7 +381,9 @@ dependencies {
     // Because RxAndroid releases are few and far between, it is recommended you also
     // explicitly depend on RxJava's latest version for bug fixes and new features.
     implementation 'io.reactivex.rxjava2:rxjava:2.2.19'
-    implementation 'com.evernote:android-job:1.2.6'
+    implementation('com.github.devv911:android-job:1.4.5') {
```

Comment:
Create an issue to remove this and either update the client-core version or use an official version from evernote-job

Reply:
created issue for it  https://github.com/opensrp/support/issues/298

Reply:
I think the issue should be to remove this dependency with an official one or use client-core v6